django/api/views.py

```diff
@@ -123,7 +123,7 @@ def supported_boards(self, request, pk=None):
         return Response(serializer.data)
 
 
-    @detail_route(methods=['GET'], permission_classes=[IsAdminUser, ])
+    @detail_route(methods=['GET'], permission_classes=[permissions.AllowAny, ])
```

Comment:
Why this permission change? Is everyone able to download, no matter if logged in or not?

Reply:
The builder has to fetch the app somewhere. This should change when we have an API gateway